api/queries_issue.go

```
@@ -167,7 +189,10 @@ func IssueCreate(client *Client, repo *Repository, params map[string]interface{}
 		"repositoryId": repo.ID,
 	}
 	for key, val := range params {
-		inputParams[key] = val
+		switch key {
+		case "assigneeIds", "body", "issueTemplate", "labelIds", "milestoneId", "projectIds", "repositoryId", "title":
```

Nit: could this `switch` statement also contain clauses for `projectV2Ids` (even if it's a no-op and will be handled below) and a `default:` clause that raises a descriptive error? Because before, unrecognized `params` keys would result in a runtime error (due to being forwarded to the API query), and now they are silently ignored.